packages/synapse-react-client/src/components/dataaccess/ReviewerDashboard.tsx

```diff
@@ -58,81 +58,96 @@ export function ReviewerDashboard(props: ReviewerDashboardProps) {
   const hasReviewerPermissions =
     userBundle?.isACTMember || userBundle?.isARReviewer
 
-  const Router = useCallback(
-    (props: MemoryRouterProps | BrowserRouterProps) => {
-      if (useMemoryRouter) {
-        return <MemoryRouter {...props} />
-      } else {
-        return <BrowserRouter {...props} />
-      }
-    },
-    [useMemoryRouter],
+  const routes: RouteObject[] = useMemo(
+    () => [
+      {
+        path: '/',
+        element: (
+          <div className="ReviewerDashboard">
+            <div className="Tabs" role="tablist">
+              {hasActPermissions && (
+                <LinkTab href="/AccessRequirements" icon="accessClosed">
+                  Access Requirements
+                </LinkTab>
+              )}
+              {hasReviewerPermissions && (
+                <LinkTab href="/Submissions" icon="discussion">
+                  Submissions
+                </LinkTab>
+              )}
+              {hasReviewerPermissions && (
+                <LinkTab href="/UserAccessHistory" icon="history">
+                  User Access History
+                </LinkTab>
+              )}
+            </div>
+            <div className="TabContentContainer">
+              <SynapseErrorBoundary>
+                <Outlet />
+              </SynapseErrorBoundary>
+            </div>
+          </div>
+        ),
```

Comment:
Top level 'layout' route. Children with matching routes will be rendered into the `Outlet`